packages/flutter_tools/lib/src/runner/flutter_command.dart

```diff
@@ -655,14 +656,18 @@ abstract class FlutterCommand extends Command<void> {
 
   /// Enables support for the hidden options --extra-front-end-options and
   /// --extra-gen-snapshot-options.
-  void usesExtraDartFlagOptions() {
+  void usesExtraDartFlagOptions({ @required bool verboseHelp }) {
     argParser.addMultiOption(FlutterOptions.kExtraFrontEndOptions,
+      help: '??? Mysterious undocumented feature. If you can figure out what this does, '
```

Comment:
Additional command line arguments that will be passed directly to the dart front end. i.e. `--enable-experiment=nonfunction-type-aliases`